### DISCUSSION TITLE
Unable to connect to backend when sent an request apiman endpoint

### DISCUSSION THREAD

**Discussion by KwanchaiJc** [apiman/apiman#2152] [Help] [answered]
wildfly-23.0.2.Final

1.I've created simple api in apiman but when i tried to sent request after apiman generate apiendpoint it's failed i don't know what error mean. Just local env.
2.i can't access apimanui via local ip:8080/apimanui/ , i have to access via localhost:8080/apimanui only.  How can i access via ip ?

![image](https://user-images.githubusercontent.com/85068934/176593190-2fea303d-08ca-4461-9c65-de57c8fa9639.png)

![image](https://user-images.githubusercontent.com/85068934/176593256-92d0634b-66d6-4ff8-a52b-7280d0cbb76e.png)

![image](https://user-images.githubusercontent.com/85068934/176593289-28aa8ac9-c48d-4b79-8b93-38af9f49cbf0.png)

**Comment by volkflo** [comment, marked as answer]
If I remember correctly then wildfly defaults to localhost if it is not configured different. 
Try `./bin/standalone.sh -b  0.0.0.0 -bmanagement 0.0.0.0` to listen on all interfaces.

Second here simply means that the apiman gateway cannot reach the backend API (your unmanaged API).
Make sure your API is reachable (firewall, network, ip, ports, dns etc...)
As always please provide logs instead of screenshots of logs.
